docker-compose.yml

```diff
@@ -0,0 +1,46 @@
+services:
+  db:
+    container_name: postgres_container
+    image: postgres
+    restart: always
+    ports:
+      - "5432:5432"
+    environment:
+      POSTGRES_USER: app_user
+      POSTGRES_PASSWORD: app_password
+      POSTGRES_DB: cyber_mysticism
+    healthcheck:
+      test: [ "CMD-SHELL", "pg_isready -U app_user -d cyber_mysticism" ]
+      interval: 10s
+      timeout: 5s
+      retries: 5
+    volumes:
+      - pgdata:/var/lib/postgresql/data
+
+  three-cards-divination:
+    container_name: three_cards_divination_container
+    image: ${SERVICE_IMAGE:-aledobrean/three-cards-divination:latest}
```

Comment:
**Suggestion:** Consider adding a `restart` policy to the `three-cards-divination` service. This will ensure that the service automatically restarts if it crashes or if the server reboots. [enhancement]
```suggestion
  three-cards-divination:
    container_name: three_cards_divination_container
    image: ${SERVICE_IMAGE:-aledobrean/three-cards-divination:latest}
    restart: always
```